internal/templates/dashboard.go.html

```diff
@@ -10,6 +10,13 @@
         <i class="fa fa-gear"></i>
     </a>
 </div>
+
+<div>
+    <form action="/logout" method="post">
+        <button class="btn" type="submit">Log out</button>
+    </form>
```

Comment:
We shouldn't be using a form because there is no data being passed in through this "form" just make it a regular button.